tests/lib/qunit-assert-domequal.js

```diff
@@ -113,6 +135,52 @@ function extract( selector, message ) {
 	} );
 	result.style = getElementStyles( elem[ 0 ] );
 	result.events = $._data( elem[ 0 ], "events" );
+
+	// jQuery >=3.4.0 uses a special focus/blur handler pair
+	// needed to fix various issues with checkboxes/radio buttons
+	// as well as being able to pass data in focus triggers.
+	// However, this leaves dummy focus & blur events if any of these
+	// events were ever listened to at a particular element. There's not
+	// a lot UI can do to fix this so just skip these handlers for
+	// data comparisons in tests.
+	// See https://github.com/jquery/jquery/issues/4496
+	if ( result.events && jQueryVersionSince( "3.4.0" ) ) {
+		var i, eventDataList, eventData;
+		$.each( [ "focus", "blur" ], function( index, eventType ) {
+			if ( !result.events[ eventType ] ) {
+				return;
+			}
+
+			// We're going to do a shallow copy of the event list to
+			// avoid mutating internal jQuery structures when removing
+			// extraneous focus/blur handlers.
+			result.events[ eventType ] = result.events[ eventType ].slice();
```

Comment:
Could we eliminate the following loop by doing the work here?
```suggestion
			// Only the special internal handlers
			// have the namespace field set to boolean `false`;
			// filter them out.
			result.events[ eventType ] = result.events[ eventType ].filter( function( eventData ) {
				return eventData.namespace !== false;
			} );
```

Reply:
Ha, you're right! So much simpler.